<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Seifenrechner</class>
 <widget class="QMainWindow" name="Seifenrechner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cmdZutaten">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zutatenliste
öffnen</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbFette">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>230</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblFette">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fette / Öle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdFettHnzufuegen">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hinzufügen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtFettMenge">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblFettName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFettMenge">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Menge in Gramm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name der Seife</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSeifenName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdBerechnen">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Berechnen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdLoeschen">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>460</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Löschen</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstRezept">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>271</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtAetherischOele">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbAetherischeOele">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>230</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdAethOelHinzufuegen">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hinzufügen</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAetherischeOele">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ätherische Öle</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTonerden">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tonerden</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtTonerden">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbTonerden">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>230</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdTonerdenHinufuegen">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hinzufügen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtParfuemOele">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTonerden_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tonerden</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbParfuemOele">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>230</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdParfuemOelHinzufuegen">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hinzufügen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdKraeuterHinzufuegen">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hinzufügen</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbKraeuter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>230</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTonerden_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kräuter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtKraeuter">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
